robber/helper.py

```diff
@@ -26,3 +26,41 @@ def build_called_params_string(call_args):
         match = re.search('call\((.*)\)', str(call_args))
         params = match.group(1)
         return params or 'no arguments'
+
+    @classmethod
+    def unicode_to_str(cls, obj):
```

Comment:
We don't have one. I just think that the function is shorter, less boilerplate. And if you want to import it, import the module is good enough. Writing class methods are unnecessary IMO
```python
import helper

helper.unicode_to_str(obj)
```